project/Settings.scala

```diff
@@ -33,6 +34,8 @@ object Settings {
 
     exportJars := true,
 
+    coverageEnabled := false,
```

Comment:
We need to disable coverage for Java projects to avoid this [issue](https://github.com/scoverage/sbt-scoverage/issues/275). Java projects are not meant to have coverage in any case.

Reply:
I thought that this PR was doing that: https://github.com/fauna/faunadb-jvm/pull/216/files 
Its supposed to run only for Scala 2.12. We can explicitly disable for Java project, tho.

Reply:
Mhm, from my understanding and the tests I've made locally and on CircleCI, previous configuration allowed the coverage task for all Scala versions and was also including the Java projects (it didn't output anything for them since they do not contain Scala code, but it looks like the task was taking them into consideration in any case).

The coverage for Scala 2.11 was not working since it was including the Java projects. They need to be explicitly excluded as explained [here](https://github.com/scoverage/sbt-scoverage/issues/275)! :)

Reply:
ok.. just being sure its working for both scala 2.11 and 2.12, and this PR will be good to go